src/main/java/com/dongtronic/diabot/platforms/discord/listeners/HelpListener.kt

```diff
@@ -31,7 +45,9 @@ class HelpListener : Consumer<CommandEvent> {
             // Show extended help card
             buildSpecificHelp(embedBuilder, allCommands, event)
             try {
-                event.author.openPrivateChannel().queue{channel -> channel.sendMessage(embedBuilder.build()).queue()}
+                event.author.openPrivateChannel().queue {
```

Comment:
I think instead of `.queue()` we can use `.submit()` which has nicer support for error handling.

Reply:
I've changed over to using `.submit()`. I'm not fully aware of the documentation for this so please let me know if I've made any mistakes.